docs/en/ingest-management/fips-ingest.asciidoc

```diff
@@ -0,0 +1,119 @@
+[role="xpack"]
+[[fips-ingest]]
+= FIPS mode for Ingest tools
+
+preview::[]
+
+{agent}, {fleet}, {filebeat}, {metricbeat}, and APM Server binaries are built and are configured to use FIPS 140-2 compliant cryptography.
+Generally speaking FIPS 140-2 requirements can be summarized as:
+
+- linking against a FIPS certified cryptographic library
+- using only FIPS approved cryptographic functions
+- ensuring that the configuration of the component is FIPS 140-2 compliant.
+
+[discrete]
+[[fips-binaries]]
+== FIPS-compatible binaries and configuration 
+
+FIPS compatible binaries for {agent}, {fleet}, {filebeat}, {metricbeat}, and APM Server are available for link:https://www.elastic.co/downloads[download].
+Look for the `Linux 64-bit (FIPS)` or `Linux aarch64 (FIPS)` platform option on the product download pages for {agent} and {fleet}, {filebeat}, and {metricbeat}.
+Look for the `Linux x86_64 (FIPS)` or `Linux aarch64 (FIPS)` platform option on the APM Server download page.
```

Comment:
These read like discrete instructions. I wonder if they could be structured like a list.